webapp/src/libs/hooks/useChat.ts

```diff
@@ -169,11 +165,19 @@ export const useChat = () => {
                         botResponseStatus: undefined,
                         userDataLoaded: false,
                         disabled: false,
+                        hidden: !features[FeatureKeys.MultiUserChat].enabled && chatUsers.length > 1,
```

Comment:
I'm not sure I get the logic...  We hide this if multiuser is enabled or if chatUsers.length is equal to 1?

Reply:
We hide chats that have multiple users only when the feature is not enabled.